### PR TITLE
Added test to class Town

### PR DESCRIPTION
**Pull request by lechemon4000**
The structure of this class is done but it could be improved and we want to know your opinions or suggestions. Thanks.

### CODE REVIEW

File: src/main/java/org/fundacionjala/at15/pokemon/Town.java [hunk 1]
```diff
@@ -7,32 +7,48 @@
...
+
     public Store getStore() {
         return store;
     }
 
+    public Gym getGym() {
+        return gym;
+    }
+
+    public ArrayList<Pokemon> getPokemonsOfTheTown() {
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/pokemon/Town.java:46]: to be consistent with the attribute's name, this method could be getWildPokemons()

File: src/main/java/org/fundacionjala/at15/pokemon/Town.java [hunk 2]
```diff
@@ -7,32 +7,48 @@
...
+        return wildPokemons;
+    }
+
+    public ArrayList<Trainer> getTrainersOfTheTown() {
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/pokemon/Town.java:50]: to be consistent with the attribute's name, this method could be getTrainers()

File: src/test/java/org/fundacionjala/at15/pokemon/TownTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    @Test
+    public void itShouldAddACenterToTown() {
+        Town paleta = new Town();
+        Center pokemonCenter = new Center();
+        paleta.getStore().getClass();
+
+        assertEquals(pokemonCenter.getClass(), paleta.getCenter().getClass());
+    }
+    @Test
+    public void itShouldAddAToGymTown() {
```
Comment (adhes14) [inline src/test/java/org/fundacionjala/at15/pokemon/TownTest.java:47]: check this name

File: src/main/java/org/fundacionjala/at15/pokemon/Town.java [hunk 4]
```diff
@@ -7,32 +7,48 @@
...
-        // center = new Center();
-        // gym = new Gym();
+        center = new Center();
+        gym = new Gym();
     }
 
     public void addWildPokemonsToTown(int wildPokemonsTotal) {
         for (int ind = 0; ind < wildPokemonsTotal; ind++) {
-            wildPokemons.add(new Pokemon(POKEMON_HP, "Pokemon" + ind));
+            wildPokemons.add(ind, new Pokemon(POKEMON_HP, "Pokemon"));
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/pokemon/Town.java:23]: Could be only:
`wildPokemons.add(new Pokemon(POKEMON_HP, "Pokemon"));`

File: src/main/java/org/fundacionjala/at15/pokemon/Town.java [hunk 5]
```diff
@@ -7,32 +7,48 @@
...
         }
     }
 
     public void addTrainersToTown(int trainersTotal) {
         for (int ind = 0; ind < trainersTotal; ind++) {
             Pokemon pokemon = new Pokemon(POKEMON_HP, "Pikachu");
-            trainers.add(new Trainer(pokemon, "Trainer" + ind));
+            trainers.add(ind, new Trainer(pokemon, "Trainer"));
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/pokemon/Town.java:30]: Could be only:
`trainers.add(new Trainer(pokemon, "Trainer"));`

**Review by adhes14 — COMMENTED** [review]
It looks good, only some small changes to be changed.

File: src/main/java/org/fundacionjala/at15/pokemon/Town.java (hunk 1 above)
Comment (lechemon4000) [reply src/main/java/org/fundacionjala/at15/pokemon/Town.java:46]: Thanks Adhemar we will change the names to be more consistents

**Review by Dru27 — APPROVED** [review]

**Review by adrianovi — APPROVED** [review]

**Comment by adrianovi** [comment]
Good job

**Pull request merged**